src/devices/Bmi088/Bmi088.cs

```diff
@@ -0,0 +1,301 @@
+        /// </summary>
+        public AccelerometerScaleRange AccelerometerScaleRange
+        {
+            get => _accelerometerScaleRange;
+
+            set
+            {
+                if (value == AccelerometerScaleRange.Range3g)
+                {
+                    _accelerometerMultiplier = 3000 / 32768f;
+                }
+                else if (value == AccelerometerScaleRange.Range6g)
+                {
+                    _accelerometerMultiplier = 6000 / 32768f;
+                }
+                else if (value == AccelerometerScaleRange.Range12g)
+                {
+                    _accelerometerMultiplier = 12000 / 32768f;
+                }
+                else if (value == AccelerometerScaleRange.Range24g)
+                {
+                    _accelerometerMultiplier = 24000 / 32768f;
+                }
+
+                WriteByte(_i2cAccelerometer, (byte)AccelerometerRegister.Range, (byte)value);
+                _accelerometerScaleRange = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the sensitivity of the gyroscope (125, 250, 500, 1000 or 2000 degrees/second)
+        /// </summary>
+        public GyroscopeScaleRange GyroscopeScaleRange
+        {
+            get => _gyroscopeScaleRange;
+
+            set
+            {
+                if (value == GyroscopeScaleRange.Range2000)
+                {
+                    _gyroscopeMultiplier = 2000 / 32768f;
+                }
+                else if (value == GyroscopeScaleRange.Range1000)
+                {
+                    _gyroscopeMultiplier = 1000 / 32768f;
+                }
+                else if (value == GyroscopeScaleRange.Range500)
+                {
+                    _gyroscopeMultiplier = 500 / 32768f;
+                }
+                else if (value == GyroscopeScaleRange.Range250)
+                {
+                    _gyroscopeMultiplier = 250 / 32768f;
+                }
+                else if (value == GyroscopeScaleRange.Range125)
+                {
+                    _gyroscopeMultiplier = 125 / 32768f;
+                }
+
+                WriteByte(_i2cGyroscope, (byte)GyroscopeRegister.Range, (byte)value);
+                _gyroscopeScaleRange = value;
+            }
+        }
+
+        /// <summary>
+        /// Resets the accelerometer
+        /// </summary>
+        protected void ResetAccelerometer()
+        {
+            WriteByte(_i2cAccelerometer, (byte)AccelerometerRegister.SoftReset, 0xB6);
+        }
+
+        /// <summary>
+        /// Resets the gyroscope
+        /// </summary>
+        protected void ResetGyroscope()
+        {
+            WriteByte(_i2cGyroscope, (byte)GyroscopeRegister.SoftReset, 0xB6);
+        }
+
+        /// <summary>
+        /// Id of the accelerometer
+        /// </summary>
+        /// <returns></returns>
+        protected byte GetAccelerometerId()
```

Comment:
I asked in the previous review is this can be a Version? And if this is a specific number that is always the same?
In that case, would make sense to check them at initialization.